cvmfs/cvmfs.cc

```diff
@@ -182,6 +182,7 @@ lru::InodeCache *inode_cache_ = NULL;
 lru::PathCache *path_cache_ = NULL;
 lru::Md5PathCache *md5path_cache_ = NULL;
 glue::InodeTracker *inode_tracker_ = NULL;
+OptionsManager *options_manager_ = NULL;

```

Comment:
Yes, it is. The first use is in the line 1834 of the cvmfs.cc file. After that it's also used many times.